gammapy/data/tests/test_data_store.py

```diff
@@ -88,6 +88,12 @@ def test_datastore_get_observations(data_store):
     observations = data_store.get_observations([11111, 23523], skip_missing=True)
     assert observations[0].obs_id == 23523
 
+    # Test that only desired IRFs are loaded
+    observations = data_store.get_observations([23523], required_IRF=["aeff"])
```

Comment:
This behaviour I find a bit strange: so specifying `required_IRF=["aeff"]` prevents users from loading the background even if it is available? I think the behaviour should be such, that one just skips the run if a required IRFs is not present instead...

Reply:
I tried to think about an optimal behaviour - the way I see it is this is the list of IRFs that will be used in the analysis. Since the other IRFs will not be used, they are not loaded.
Would you prefer to load the non-essential IRFs as well? Will adapt accordingly then.

Currently, one does skip the run if a required IRF is not present.

Reply:
Yes, I think I prefer to allow to load the non-required IRFs as well. Forbidding this I think is too restrictive for no good reason. My preferable behaviour would be just to have the option in `.get_observations(..., skip_missing_irfs=[])` or `get_observations(..., required_irfs=[])` .